src/perf/lib/HpsClient.cpp

```diff
@@ -72,7 +72,8 @@ HpsClient::Init(
     }
 
     const char* target;
-    if (!TryGetValue(argc, argv, "target", &target)) {
+    if (!TryGetValue(argc, argv, "target", &target) &&
+        !TryGetValue(argc, argv, "server", &target)) {
```

Comment:
Why this change? We don't specify "server" is allowed anywhere in the help, and this seems completely unrelated.

Reply:
This is a improvement for my personal well-being. 😄 I can never remember the arg name, and I always end up using `-server`. I don't feel the need to clutter the help, but I'd like to support it.